spec/acceptance/snmp_spec.rb

```diff
@@ -120,4 +123,25 @@ class { 'snmp':
       it { is_expected.to be_running }
     end
   end
+
+  context 'with modified snmpd options' do
+    it 'executes with modified options' do
+      pp = %(
+        class { 'snmp':
+          snmpd_config => [
+            'rouser myops authPriv -V all_view',
+            'rouser mysubcontractor authPriv -V custom_view',
+          ],
+          snmpd_options => '-a -u Debian-snmp -g Debian-snmp -I -smux mteTrigger mteTriggerConf -f',
```

Comment:
Oups, we have to call the class with modified options in `snmpd_options`
```suggestion
          snmpd_options => '#{modified_snmpd_options}',
```